tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -17,6 +17,78 @@ Documentation  Test 5-4 - High Availability
 Resource  ../../resources/Util.robot
 Suite Teardown  Nimbus Cleanup  ${list}
 
+*** Keywords ***
+Check VM Info
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.info \*
+    Should Be Equal  ${rc}  0
+    Log To Console  ${output}
```

Comment:
Here we want to list all the vms whereas the existing `Get VM Info` keyword takes `${vm}` as input. I think it is more convenient to use `govc vm.info \*` here. And we would remove these additional logs after we fix this issue.